mep/accounts/xml_models.py

```diff
@@ -364,29 +392,33 @@ def to_db_event(self, account):
             # *should* already exist from regularized title import;
             # if item does not exist, create a new stub record
             borrow.item, created = Item.objects.get_or_create(mep_id=self.item.mep_id)
-            if created:
-                borrow.item.title = self.item.title
-
-            bibl_info = []
-            # if bibliographic details are present, document them in the item note
-            # to support book title data work
-            if self.item.author and self.item.author not in borrow.item.notes:
-                bibl_info.append('Author: %s' % self.item.author)
-            if self.item.publisher and self.item.publisher not in borrow.item.notes:
-                bibl_info.append('Publisher: %s' % self.item.publisher)
-            if self.item.date and self.item.date not in borrow.item.notes:
-                bibl_info.append('Date: %s' % self.item.date)
-            # TODO: might want to put volume/issue on borrowing event rather than item
-            for bibl_scope in self.item.scope_list:
-                # e.g. number/issue and any text...
-                bibl_info.append('%s %s' % (bibl_scope.unit, bibl_scope.text))
-
-            if bibl_info:
-                borrow.item.notes += '\n'.join(bibl_info)
-
-            # save the item if it is new or notes were added
-            if bibl_info or created:
-                borrow.item.save()
+
+        # if no mep id, we still want to create a stub record
+        else:
+            borrow.item = Item.objects.create()
+            created = True
+
+        # if item was newly created OR borrow title is unclear, set the title
+        if created or self.item.title.is_unclear:
```

Comment:
I hit `AttributeError: 'NoneType' object has no attribute 'is_unclear'` here. Do we need to handle `None` title?

Reply:
Ah. Yes we do!